readme.md

```diff
@@ -506,6 +506,7 @@
 - [JSON](https://github.com/burningtree/awesome-json#readme) - Text based data interchange format.
 	- [GeoJSON](https://github.com/tmcw/awesome-geojson#readme)
 	- [Datasets](https://github.com/jdorfman/awesome-json-datasets#readme)
+- [CSV](https://github.com/secretGeek/awesomeCSV#readme) - The most futuristic data format from the distant past.
```

Comment:
The description should describe what it is. This sounds more like a marketing tagline.

Reply:
Added new commit which updates this description to "Comma-Separated-Text, an ancient format that remains widely used."

Reply:
That still doesn't describe **what** it is... 

You could have just copy-pasted something from Wikipedia: https://en.wikipedia.org/wiki/Comma-separated_values

I would recommend something like this:

> A text file format that stores tabular data and uses a comma to separate values.

Reply:
I went with exactly that, thank you.